Assets/Engine/SeedAssetList.seed

```diff
@@ -232,14 +224,6 @@
       <Class name="unsigned int" field="platformFlags" value="255" type="{43DA906B-7DEF-4CA8-9790-854106D3F983}"/>
       <Class name="AZStd::string" field="pathHint" value="scripts/enginecommon.luac" type="{03AAAB3F-5C47-5A66-9EBC-D5FA4DB353C9}"/>
```

Comment:
Can we remove this one too? I believe I've deleted this in the redcode change https://github.com/aws-lumberyard-dev/o3de/commit/adc2f3f14d123db215da57fd212b32278175342f#diff-4e5ea674abf1a32fc45dbe021f8052fb955314bd3858023400730febf38b1878

Reply:
the .luac file is the compiled version of the .lua file and should no longer get generated or copied over

Reply:
I just merged latest, I think you did already remove it there.